spec/lib/gettext_setup_spec.rb

```diff
@@ -50,8 +52,10 @@
       expect(GettextSetup.default_locale).to eq('en')
       expect(GettextSetup.candidate_locales).to include('en')
       GettextSetup.clear
+      before_locale = ENV['LANG']
       ENV['LANG'] = 'de_DE'
       expect(GettextSetup.candidate_locales).to eq('de_DE,de,en')
+      ENV['LANG'] = before_locale
```

Comment:
I think a better solution actually is just to add `LANG=en_US` before every msg* command we run.